### ISSUE TITLE
[Screen Readers-Morgan Stanley- Login page]: NVDA does not announce any information of error message which appears after activating 'Request Account Activation' button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14126]
### Steps to reproduce:
Step 1: Enable NVDA and Open the above URL in the edge Chromium browser.
Step 2: Morgan Stanley Login page will Open.
Step 3: Navigate to ‘Open Account' link and activate it.
Step 4: ‘Request Account Activation’ page will open.
Step 5: Navigate to radio buttons. Leave them unchecked.
Step 6: Navigate to continue button and activate it.
Step 7: Verify that NVDA does not announce any information of error message which appears after activating 'continue' button or not.

### Actual behavior:
NVDA does not announce any information of error message which appears after activating 'continue' button.

### Expected behavior:
NVDA should announce proper information of error message which appears after activating 'continue' button.

### System configuration
OS: WIN 11 21H2 (OS Build 22000.856)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
nvda_2022.2

#### Windows version:
Windows 11 Enterprise

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

### Other questions
N/A
#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

###Attachment:
https://user-images.githubusercontent.com/93735775/189543179-738fe99c-0af2-4d76-be39-a1bc1153c5fc.mp4

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

**Comment by Brian1Gaff** [comment]
I have seen this sort of thing on many forms where its complaining that you 
have not entered something or done so correctly. Its like it ignores the 
message, and the only way to be sure its there normally is to go to browse 
mode and look for it. I always thought this was due to the pages all doing 
something wrong in indicating where the error text was.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "msftedad" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Sunday, September 11, 2022 7:26 PM
Subject: [nvaccess/nvda] [Screen Readers-Morgan Stanley- Login page]: NVDA 
does not announce any information of error message which appears after 
activating 'Request Account Activation' button. (Issue #14126)


### Steps to reproduce:
Step 1: Enable NVDA and Open the above URL in the edge Chromium browser.
Step 2: Morgan Stanley Login page will Open.
Step 3: Navigate to ‘Open Account' link and activate it.
Step 4: ‘Request Account Activation’ page will open.
Step 5: Navigate to radio buttons. Leave them unchecked.
Step 6: Navigate to continue button and activate it.
Step 7: Verify that NVDA does not announce any information of error message 
which appears after activating 'continue' button or not.


### Actual behavior:
NVDA does not announce any information of error message which appears after 
activating 'continue' button.
-->

### Expected behavior:


### NVDA logs, crash dumps and other attachments:

### System configuration
OS: WIN 11 21H2 (OS Build 22000.856)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
nvda_2022.2

#### Windows version:
Windows 11 Enterprise

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

### Other questions
N/A
#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their 
behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing 
Tool in NVDA's tools menu?
Yes

###Attachment:
https://user-images.githubusercontent.com/93735775/189543179-738fe99c-0af2-4d76-be39-a1bc1153c5fc.mp4



-- 
Reply to this email directly or view it on GitHub:
https://github.com/nvaccess/nvda/issues/14126
